### PR TITLE
Fix F3 invariant

### PR DESCRIPTION
**Pull request by grandizzy**
- if prefixSum is deposit up to index then bucket index from deposit should be prefixSum + 1

### CODE REVIEW

**Review by EdNoepel — APPROVED** [review]
LGTM

File: tests/forge/ERC20Pool/invariants/BasicInvariants.t.sol
Comment (mattcushman) [inline tests/forge/ERC20Pool/invariants/BasicInvariants.t.sol:369]: In theoryI think that this should pass (if we replace > with >= which is in the code anyway) but looks perhaps like there is some rounding issue here.  I'm not to concerned about it if it is indeed a rounding issue.

**Review by mattcushman — APPROVED** [review]
While mathematically i think it should be true without the +1, I think it's OK to have the +1 and should not have meaningful consequences for the contract (basically, in some extreme cases where the cumulative deposit is close to the debt, it will slightly affect the LUP computation, but things like the exact timestamps for interest accrual already have similar effects).

**Comment by grandizzy** [comment]
> 

thanks, won't merge this one right now but will do if there are more tests failing

**Pull request closed without merging**
